blocks/sketch/src/controls.js

```diff
@@ -30,51 +37,82 @@ const brushes = [
 	},
 ];
 
-const Controls = ( { clear, color, setColor, preset, setPreset, isEmpty } ) => {
+const Controls = ( {
+	clear,
+	color,
+	setColor,
+	preset,
+	setPreset,
+	isEmpty,
+	title,
+	setTitle,
+} ) => {
 	const colors = useSetting( 'color.palette' ) || [];
 	return (
-		<BlockControls group="block">
-			<ToolbarDropdownMenu
-				isCollapsed
-				popoverProps={ {
-					className: 'wp-block-a8c-sketch__brush-style-popover',
-					isAlternate: true,
-				} }
-				icon={ <Icon icon={ BrushSizeControlIcon } /> }
-				label={ __( 'Brush', 'sketch' ) }
-				controls={ brushes.map( ( control ) => ( {
-					...control,
-					isActive: control.value === preset,
-					onClick: () => {
-						if ( control.value !== preset ) {
-							setPreset( control.value );
+		<>
+			<BlockControls group="block">
+				<ToolbarDropdownMenu
+					isCollapsed
+					popoverProps={ {
+						className: 'wp-block-a8c-sketch__brush-style-popover',
+						isAlternate: true,
+					} }
+					icon={ <Icon icon={ BrushSizeControlIcon } /> }
+					label={ __( 'Brush', 'sketch' ) }
+					controls={ brushes.map( ( control ) => ( {
+						...control,
+						isActive: control.value === preset,
+						onClick: () => {
+							if ( control.value !== preset ) {
+								setPreset( control.value );
+							}
+						},
+					} ) ) }
+				/>
+				<ToolbarDropdownMenu
+					isCollapsed
+					popoverProps={ { isAlternate: true } }
+					icon={
+						<Icon icon={ <ColorControlIcon color={ color } /> } />
+					}
+					label={ __( 'Color', 'sketch' ) }
+				>
+					{ () => (
+						<ColorPalette
+							clearable={ false }
+							colors={ colors }
+							color={ color }
+							disableCustomColors={ true }
+							onChange={ setColor }
+						/>
+					) }
+				</ToolbarDropdownMenu>
+				<ToolbarButton
+					icon={ trash }
+					onClick={ clear }
+					label={ __( 'Clear canvas', 'sketch' ) }
+					disabled={ isEmpty }
+				/>
+			</BlockControls>
+			<InspectorControls>
+				<PanelBody title={ __( 'Sketch settings' ) }>
+					<TextareaControl
+						label={ __( 'Title' ) }
+						value={ title }
+						onChange={ setTitle }
+						help={
+							<>
+								<ExternalLink href="https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title">
```

Comment:
Let's remove the external link and rewrite the text to be more about "why" you should add a description

Reply:
Updated in bd371cf for _Add a short-text description so it's recognized as the accessible name for the sketch._

Reply:
Maybe `A short-text description will be recognized as the accessible name for the sketch` ?